cdap-docs/reference-manual/source/http-restful-api/metadata.rst

```diff
@@ -578,12 +578,35 @@ Since CDAP also annotates *system* metadata to entities by default as mentioned
   ``mapreduce:<mapreduce-name>``, ``spark:<spark-name>``, ``worker:<worker-name>``,
   ``workflow:<workflow-name>``
 
-- datasets, streams or views with the specified field:
+- datasets, streams or views with schema field:
+
   - field name only: ``<field-name>``
-  - field type only: ``<field-type>``
-  - field name with a particular type: ``<field-name>:<field-type>``
+  - field name with a type: ``<field-name>:<field-type>``, where ``field-type`` can be
+
+    - simple types - ``int``, ``long``, ``boolean``, ``float``, ``double``, ``bytes``, ``string``, ``enum``
+    - ``array``, ``map``, ``record`` or ``union``
+
+::
+
+  {
+     "type":"record",
+     "name":"employee",
+     "fields":[
+        {
+           "name":"employeeName",
+           "type":"string"
+        },
+        {
+           "name":"departments",
+           "type":{
+              "type":"array",
+              "items":"long"
+           }
+        }
+     ]
+  }
 
-Searches are case-insensitive.
+With a schema as shown above, queries such as ``employee:record``, ``employeeName:string``, ``departments``, ``departments:array`` can be issued.
```

Comment:
lets break this line to ~120 chars?
Also, should we add an example with multiword search queries?